packages/forms-web-app/src/views/selected-appeal/appeal-details.njk

```diff
@@ -8,15 +8,6 @@
 
 {% set title="Appeal details " + appeal.appealNumber + " - " + titleSuffix + " - GOV.UK" %}
 
-	{% block head %}
-		{% if cssOverride %}
-			<style {% if cspNonce %}nonce={{ cspNonce }}{% endif %}>{{ cssOverride | safe }}</style>
-			<style {% if cspNonce %}nonce={{ cspNonce }}{% endif %}>.govuk-template { background-color: #FFFFFF; }</style>
-		{% else %}
-			{{ super () }}
```

Comment:
do we still need this super call or is it redundant?

Reply:
it is redundant, I put it in before because putting the head block in this njks file overrode the default super, so it needed to be restated explicitly here.